### ISSUE TITLE
Running the App on Windows causes Microsoft Edge to get pinned to the taskbar

### ISSUE THREAD

**Issue by KMikeeU** [s77rt/hashcat.launcher#86]
I downloaded the latest release and moved the .exe to my programs folder, which contains hashcat. Upon launching hashcat.launcher, Microsoft Edge is pinned to my taskbar after just a few seconds, and remains there after closing hashcat.launcher. Edge does not actually open, but I am assuming it's the default browser engine you are using for Windows?

**Comment by s77rt** [comment]
thanks for the report
i have not tested the app with edge as the default engine, it would usually use Chrome if installed
will need to make more tests on Edge and confirm this issue

besides edge not getting closed, i assume the app is working fine, right?

**Comment by KMikeeU** [comment]
Yes, the rest is working fine, :) I'm assuming it's probably just a Microsoft thing forcing edge on users?

**Comment by LLH-l** [comment]
Using Chrome does not automatically translate to other national languages
Using edge can automatically translate into other national languages, but some function translation pages cannot be displayed normally, and the content is empty

Hope to follow up on this issue, because it can support translation into many countries' languages
